### DISCUSSION TITLE
Java o Python para el desarrollo de APIs

### DISCUSSION THREAD

**Discussion by Carlo-MG** [community/community#125451] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Estoy interesado en aprender a crear APIs, pero no estoy seguro si debería usar Python o Java

**Comment by davevad93** [comment, marked as answer]
Hi @Carlo-MG ,

Personally I would say `Python` I may be biased on this, but I think it's better than `Java` because the learning curve to make APIs with `Python` is smaller than doing it with `Java`, besides, there are two very good frameworks to do it: `FastAPI` and `Flask`. Having said that, it's true that `Java` can be more efficient in computational performance than `Python` but in the end it also depends on which programming language you like the most.
